### DISCUSSION TITLE
MySQL同步到Doris，同步变更没有反应.

### DISCUSSION THREAD

**Discussion by peachjuice001** [apache/flink-cdc#3617] [Q&A]
使用官网的例子的快速开始-MySQL同步到Doris-同步变更；在mysql侧insert数据未同步至Doris

**Comment by JanSay** [comment]
一样，我用的 3.2.0 版本的 flinkcdc。修改或者新增，Doris 都没有反应

**Comment by yuxiqian** [comment]
Hi, Doris sink connector has some buffering strategy inside that might cause data synchronoization lagging. Try adjusting 
`sink.buffer-flush.max-rows`, `sink.buffer-flush.max-bytes`, and `sink.buffer-flush.interval` options.
